config/formats.js

```diff
@@ -409,12 +409,11 @@ let Formats = [
 
 		mod: 'suicidecup',
 		forcedLevel: 100,
-		ruleset: ['[Gen 7] Ubers', '!Mega Rayquaza Clause', '!OHKO Clause'],
```

Comment:
Apparently the `!` items need to be before the ruleset, so `['!Mega Rayquaza Clause', '!OHKO Clause', '[Gen 7] Ubers']` works. (I got this right for "Real 1v1"...)

Reply:
welp